src/core/linker.cpp

```diff
@@ -107,7 +119,9 @@ void Linker::Execute() {
         }
     }
 
+    #ifdef __x86_64__
     SetTcbBase(nullptr);
```

Comment:
Instead of stubbing out `SetTcbBase` it should probably just have an ARM stub for now, it's going to need to be set up somehow for ARM translated code to access. Ditto for the other cases of this.

Reply:
Done